sunpy/map/tests/test_mapbase.py

```diff
@@ -175,6 +175,12 @@ def test_date_aia(aia171_test_map):
     assert aia171_test_map.date == parse_time('2011-02-15T00:00:00.34')
 
 
+def test_date_setter(generic_map):
+    dtime_old = generic_map.date
+    generic_map.date = '2011-02-15T00:00:00.34'
+    assert generic_map.date == parse_time('2011-02-15T00:00:00.34') != dtime_old
+
+
```

Comment:
Should we also add a negative test? What are we not going to accept? Are the tests on `parse_time` enough?

Reply:
I think parse_time is generic enough, since the reason parse_time exists is to massage a range of sensible inputs into an `astropy.time.Time`.